### PR TITLE
Recent issues

### PR DESCRIPTION
**Pull request by samozzy**
Close the various recent small issues reported

### CODE REVIEW

File: _people/sophy_baxter.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Sophy Baxter
+submitted: 2020-03-02 
+gender: female 
+# headshot: 
+course: BSc Nursing 
+graduated: 2022
```
Comment (BenWoodford76) [inline _people/sophy_baxter.md]: did we fix the fact that putting in a grad year in the future would automatically label them as graduated?

Comment (samozzy) [reply _people/sophy_baxter.md]: Yes I think so, thanks to @emmambarber 😛

**Comment by BenWoodford76** [comment]
@samozzy you happy for this to merge?

**Comment by samozzy** [comment]
@BenWoodford76 Oui

**Pull request merged**